flink-runtime/src/main/java/org/apache/flink/runtime/messages/checkpoint/TriggerCheckpoint.java

```diff
@@ -84,7 +84,11 @@ else if (o instanceof TriggerCheckpoint) {
 
 	@Override
 	public String toString() {
-		return String.format("Trigger Checkpoint %d@%d for (%s/%s)", 
-				getCheckpointId(), getTimestamp(), getJob(), getTaskExecutionId());
+		return String.format(
```

Comment:
nit: checkstyle hotfix not related to _removing legacy JobManager_.

Reply:
True, I'll revert it.

Reply:
True, I'll make it a separate commit.